clients/include/flops.hpp

```diff
@@ -720,42 +720,21 @@ constexpr double syrkx_gflop_count<hipblasDoubleComplex>(int n, int k)
 
 /* \brief floating point counts of TRSM */
 template <typename T>
-constexpr double trmm_gflop_count(int m, int n, hipblasSideMode_t side)
+constexpr double trmm_gflop_count(int m, int n, int k)
 {
-    if(HIPBLAS_SIDE_LEFT == side)
-    {
-        return (1.0 * m * n * (m + 1)) / 1e9;
-    }
-    else
-    {
-        return (1.0 * m * n * (n + 1)) / 1e9;
-    }
+    return (1.0 * m * n * (k + 1)) / 1e9;
 }
 
 template <>
-constexpr double trmm_gflop_count<hipblasComplex>(int m, int n, hipblasSideMode_t side)
+constexpr double trmm_gflop_count<hipblasComplex>(int m, int n, int k)
 {
-    if(HIPBLAS_SIDE_LEFT == side)
-    {
-        return 4 * (1.0 * m * n * (m + 1)) / 1e9;
-    }
-    else
-    {
-        return 4 * (1.0 * m * n * (n + 1)) / 1e9;
-    }
+    return 4 * (1.0 * m * n * (k + 1)) / 1e9;
 }
```

Comment:
similar to comment above, I think it is best to get trmm_gflop_count from the following code from the file http://www.netlib.org/blas/cblas3time.f 

            S1 = SECOND( )
            CALL CTRMM( SIDE, UPLO, TRANSA, 'N', M, N, ALPHA, A, LDA, B,
     $                  LDB )
            S2 = SECOND( )
            TIME = ( S2 - S1 ) - ( S1 - S0 )
            IF( SIDE.EQ.'L' )THEN
               NOP = 4*M*M*N
            ELSE
               NOP = 4*M*N*N
            END IF
            IF( TIME.LE.0.0E0 )THEN
               RESLTS( IMVAL, INVAL ) = 0.0E0
            ELSE
               RESLTS( IMVAL, INVAL ) = NOP/( 1.0E6*TIME )
            END IF

for rocBLAS the value would be 4*m*m*n/1e9 if HIPBLAS_SIDE_LEFT == side and 4*m*n*n if HIPBLAS_SIDE_LEFT != side

Reply:
Same here, I can get rid of the `+1` but I'm not sure I see any reason to change it to passing in `side` rather than `k`.